### PR TITLE
add debugging to test

### PR DESCRIPTION
**Pull request by jeffwashington**
#### Problem
when this test fails, it would be useful to know more information
#### Summary of Changes

Fixes #

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -5541,7 +5541,7 @@ dependencies = [
  "solana-rpc",
  "solana-runtime",
  "solana-sdk",
- "tokio 1.10.1",
+ "tokio 1.11.0",
```
Comment (brooksprumo) [inline Cargo.lock:5544]: Interesting. Did your changes pull in these Cargo modifications? Or maybe this was to fix a CI check?

**Review by brooksprumo — APPROVED** [review]
Looks good other than the Cargo.lock question.

File: Cargo.lock (hunk 1 above)
Comment (jeffwashington) [reply Cargo.lock:5544]: i suspect someone will update master. I didn't cause it that I can imagine.

Comment (CriesofCarrots) [reply Cargo.lock:5544]: I merged bumps to futures and tokio recently. Perhaps those prs missed bumping replica-lib, for some reason 🤔 
Anyway, shouldn't be an issue to pull in here.

**Pull request merged**